packages/experimental/Tooltip/src/useTooltip.ts

```diff
@@ -0,0 +1,10 @@
+import type { TooltipProps } from './Tooltip.types';
+
+export const useTooltip = (props: TooltipProps): TooltipProps => {
+  const { ...rest } = props;
```

Comment:
Seems like this isn't used, do we need this file?

Reply:
Removed!